cglm.pc.in

```diff
@@ -1,11 +1,11 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix="${prefix}"
+libdir="${exec_prefix}/lib"
+includedir="${prefix}/include"
```

Comment:
This is broken and breaks cmake.

(It also breaks autotools which uses the same template, but that's a side point).

Why does it break cmake? Good question. The answer is because it completely disrespects `-DCMAKE_INSTALL_LIBDIR=lib64` or `-DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu`. It assumes that:
- every path is relative to CMAKE_INSTALL_PREFIX
- every path is a hardcoded string, either "lib" or "include".

But AFAICT, the previous version of this code already handled the case where these assumptions were true, *and* also handled what Debian always does (`lib/{triplet}`), *and* handled systems where the libdir is "lib64".

And it handled this by including the *idiomatic* logic, where if `CMAKE_INSTALL_*` directories are absolute paths, it respects those, and if they aren't, it calculates them relative to CMAKE_INSTALL_PREFIX.

Reply:
@eli-schwartz thanks for the catch and investigation #272 reverts this as you suggested